Week3/homework/js-exercises/1-compliment.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+function giveCompliment(name){
+    const complimentArr = ["cool", "nice", "super", "awesome", "clever", "smart", "funny", "great", "thoughtful", "generous"];
+    let randomNumber = Math.floor(Math.random() * complimentArr.length);
+    return "You are " + complimentArr[randomNumber] + " " +  name;
```

Comment:
👍 for logic. I would like to introduce you to a new feature in JavaScript called template literals which I think you can use to improve the `return` statement of the function. Have a look here - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals